lib/rdf/ldp/helper.rb

```diff
@@ -3,14 +3,16 @@
 # mixin methods
 module RDF
   class LDP < RDF::StrictVocabulary("http://www.w3.org/ns/ldp")
+    extend Deprecation
```

Comment:
sorry - was looking at your comment before refresh -- I was following the examples in  projectblacklight/blacklight.